src/item.cpp

```diff
@@ -13957,7 +13957,8 @@ bool item::process_blackpowder_fouling( Character *carrier )
 {
     // rust is deterministic. 12 hours for first rust, then 24 (36 total), then 36 (72 total) and finally 48 (120 hours to go to XX)
     // this speeds up by the amount the gun is dirty, 2-6x as fast depending on dirt level.
-    set_var( "rust_timer", get_var( "rust_timer", 0 ) + 1 + get_var( "dirt", 0 ) / 2000 );
+    set_var( "rust_timer", get_var( "rust_timer", 0 ) + std::max( 0.3 + get_var( "dirt", 0 ) / 200,
```

Comment:
You're changing the formula, but not the comment above. Is it still accurate? Why are you changing it?

Reply:
Fixed.